### DISCUSSION TITLE
User manual

### DISCUSSION THREAD

**Discussion by NicoRio42** [karttapullautin/karttapullautin#123] [Ideas]
Karttapullautin is awesome, but it is not easy to get started with for average users (with no computer science background). A lot of people comes to me asking for help in the french orienteering community, so I [wrote a guide](https://mapant.fr/blog/karttapullautin-pour-les-nuls) (in french) for future references to stop repeating the same things all over again.

I could write an English translation without the France specific parts. If you think it can be a good idea, were should it live ? As a markdown file in the repo ? In the wiki section of the repo ? We could also consider building a documentation website even if it is a little bit more work.

**Comment by rphlo** [comment]
A Karttapullautin for dummies guide in English would be good, although your article cover only the batch mode.
If you provide an english translation, I am sure we will find a place where in can live online...

**Comment by stefankinell** [comment]
It is a bit daunting to get into how to use the settings etc in Karttapullautin, I agree. I also wrote a guide some years ago, as part of building a process/manual on how to do a automatic generated orienteering map from start to end. Some parts can be found at https://stefansolsida.wordpress.com/2020/06/22/2-laserdata-skapa-kurvor/ and overal structure at https://stefansolsida.wordpress.com/12-stegsprogrammet/

But it does not cover e.g. how to set vegetation in the right way.

A fast way here would be to link to these kind of manuals we know there are out there already. I know a handful more of them.

Another would to write a basic manual, and then have a joint wiki which can be built out when someone has time and will to do so.

**Comment by NicoRio42** [comment]
@stefankinell I didn't know about your articles, very nice! Especially your workflow for improving cliffs is very interesting.
Maybe writing another English user manual would be re-inventing the wheel, if you say there is already a handful of them out there. Like you said, maybe just linking all these resources in the README is the most pragmatic solution for now?
